src/Helpers/ApartmentHelpers.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace BuildEmpire\Apartment\Helpers;
+
+class ApartmentHelpers
+{
+    /**
+     * Does the schema only contain letters and/or numbers.
+     *
+     * @param $schemaName
+     * @return bool
+     */
+    public static function isSchemaNameValid($schemaName)
+    {
+        return (preg_match('/^[a-z][a-z0-9_]*$/', $schemaName) && !ctype_upper($schemaName));
+    }
+
+    /**
+     * This will remove any bad characters from the $schemaName variable.
+     *
+     * Note: Removes any non valid strings with #. You should be checking with isSchemaNameValid before reaching this
+     * section. This is just an extra layer of protection.
+     *
+     * @param $schemaName
+     * @return mixed
+     */
+    public static function getSchemaSafeString($schemaName) {
+        return preg_replace('/^[^0-9a-z_]+$/', '#',$schemaName);
```

Comment:
I don't like this ... not sure it's a suitable solution. Deliberately putting breaking code in as a security method is not right. 

Not happy with this, sorry.